optaplanner-core/src/main/java/org/optaplanner/core/impl/partitionedsearch/queue/PartitionChangedEvent.java

```diff
@@ -29,7 +29,7 @@
     private final int partIndex;
     private final long eventIndex;
     private final PartitionChangedEventType type;
-    private final PartitionChangeMove<Solution_> move;
+    private final transient PartitionChangeMove<Solution_> move;
```

Comment:
Moves aren't Serializable. Sprinkling transient here to make FindBugs happy but not really supporting serializing Moves has no benefit for our users.

Solution proposals
A) Add transient in places to mislead findbugs bug detection. I veto this.
B) Properly make Move's etc Serializable. I think that's unrealistic. That's pandora's box.
C) Remove Serializable from classes that findbugs correctly says aren't really serializable. I like this: no one is serializing moves/descriptors/solvers etc anyway. Yes, this can make CDI-in-the-servlet-context unhappy, but meh.
D) Sprinkle supressWarning annotations or similar Findbugs annotations to avoid findbugs complains. I don't like this much: findbugs should only add to our code in a good way.
E) Exclude the Serializable rule in the findbugs rules to check. I like this one.

Wdyt? I prefer C) or E).

Reply:
For me C) is the best options. Although E) sounds reasonably, excluding the Serializable rule does not make the classes properly Serializable either. I would say we can either do it correctly or should not do it at all.

Is there any particular situation with CDI we need to care about? That would be in my opinion the only reason for E).